modules/entrezdirect/xtract/main.nf

```diff
@@ -0,0 +1,38 @@
+process ENTREZDIRECT_XTRACT {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
If I understand it this is just text manuulation, low would be sufficient I imaigne

Reply:
True, in fact I forgot to change it.